.github/CODEOWNERS

```diff
@@ -12,10 +12,11 @@
 /src/libs/ajax/Catalog.ts @DataBiosphere/data-explorer-eng
 /src/libs/ajax/workflows-app/ @DataBiosphere/broad-workflow-management @DataBiosphere/broad-workflow-execution
 /src/pages/library/DataBrowser* @DataBiosphere/data-explorer-eng
```

Comment:
I don't see anything in `library/DataBrowser`, was that removed earlier too?

Maybe everything under `/src/pages/library` should be owned by `data-explorer-eng`, if it's not us I'm not sure who it would be.

Reply:
There is one file that includes that path (the toggle)